Vagrantfile

```diff
@@ -8,5 +8,7 @@ Vagrant.configure("2") do |config|
       apt-get update
       apt-get install -y clang-12 golang make
       update-alternatives --install /usr/bin/clang clang /usr/bin/clang-12 100
+      [ -f /usr/lib/llvm-12/bin/llvm-strip ] && [ -f /usr/local/bin/llvm-strip ] || \
+      ln -s /usr/lib/llvm-12/bin/llvm-strip /usr/local/bin/llvm-strip
```

Comment:
Could you add into the commit message any info what this is fixing?

Reply:
👍